tests/test_pickle.py

```diff
@@ -56,19 +56,19 @@ def test_accumulators(accum, args, copy_fn):
 
 
 axes_creations = (
-    (bh.core.axis._regular_uoflow, (4, 2, 4)),
-    (bh.core.axis._regular_growth, (4, 2, 4)),
-    (bh.core.axis._regular_noflow, (4, 2, 4)),
-    (bh.axis.regular_log, (4, 2, 4)),
-    (bh.axis.regular_sqrt, (4, 2, 4)),
-    (bh.axis.regular_pow, (4, 2, 4, 0.5)),
-    (bh.axis.circular, (4, 2, 4)),
-    (bh.axis.variable, ([1, 2, 3, 4],)),
-    (bh.core.axis._integer_uoflow, (1, 4)),
-    (bh.core.axis._category_int, ([1, 2, 3],)),
-    (bh.core.axis._category_int_growth, ([1, 2, 3],)),
-    (bh.core.axis._category_str, (["1", "2", "3"],)),
-    (bh.core.axis._category_str_growth, (["1", "2", "3"],)),
+    (bh.core.axis._regular_none, (4, 2, 4, None)),
```

Comment:
We'll be dropping the ability to pass metadata in as a positional argument, so we'll need to handle the final item in this tuple specially or remove it.

Reply:
The _axis are internal, they get all arguments as positional. There is no need to make their interface complicated, because the internal axis types are only called by the library.

Reply:
I want to remove the tests of the private axis types. We need to test the user-facing axis anyway and then these test are redundant.

Reply:
Okay; I don't think it's bad to test private components in unit tests, but only if they only available as an integration test using public interfaces, which is not true here. The main reason these exist was the order in which this was developed.

Reply:
Right, that makes sense.